### DISCUSSION TITLE
is mercurius-integration-testing still maintained?

### DISCUSSION THREAD

**Discussion by salzhrani** [mercurius-js/mercurius#717] [General]
it is still on [mercurius v8](https://github.com/mercurius-js/mercurius-integration-testing/blob/master/package.json#L63), and there are no PRs or answer to new issues.
would love to at least [bump the dependency](https://github.com/mercurius-js/mercurius-integration-testing/issues/3). but as I said I got no replies.

**Comment by mcollina** [comment]
Sorry about this, for some reasons I was not following the repo.
